packages/web/src/services/track-download.ts

```diff
@@ -1,15 +1,25 @@
-import { CID, TrackDownload as TrackDownloadBase } from '@audius/common'
+import { TrackDownload as TrackDownloadBase } from '@audius/common'
 
 import { audiusBackendInstance } from './audius-backend/audius-backend-instance'
 
 class TrackDownload extends TrackDownloadBase {
-  async downloadTrack(
-    cid: CID,
-    creatorNodeEndpoints: string[],
-    filename: string
-  ) {
-    const audiusLibs = await this.audiusBackend.getAudiusLibs()
-    return audiusLibs.File.downloadCID(cid, creatorNodeEndpoints, filename)
+  async downloadTrack({ url, filename }: { url: string; filename: string }) {
+    const response = await window.fetch(url)
+    if (!response.ok) {
+      throw new Error('Download unsuccessful')
+    }
+
+    const downloadURL = (url: string, filename: string) => {
```

Comment:
I'm not familiar with this code path at all, but is this really how we download tracks? make a link and calling click on it seems strange, but if it's how we do things otherwise then ok!

Reply:
yeah ive seen this a bunch in my years writing code. including in libs where i got this from, which is how current track download works. check [this](https://github.com/AudiusProject/audius-protocol/blob/main/libs/src/api/File.ts#L14) out.